media/css/firefox/all.less

```diff
@@ -1,5 +1,42 @@
 @import "../sandstone/lib.less";
 
+body.nightly {
+    color: #FFF;
+    #outer-wrapper {
+        background: url('/media/img/firefox/horizon/stars.svg') center 60px no-repeat,
```

Comment:
It might be worth adding a fallback here for browsers that don't support multiple backgrounds, else the text contrast becomes unreadable.